ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -156,6 +156,19 @@ open class NavigationBar: UINavigationBar {
         return nil
     }
 
+    @objc public func barButtonItemView(tag: Int) -> UIView? {
+        let leftBarButtonItemViews = leftBarButtonItemsStackView.arrangedSubviews
+        let rightBarButtonItemViews = rightBarButtonItemsStackView.arrangedSubviews
```

Comment:
can we just combine line 160-162 into 1 let?

Reply:
Done